Algorithms/Dynamic Programming/rod_cutting(DP).cpp

```diff
@@ -0,0 +1,54 @@
+
+#include<iostream>
+#include<bits/stdc++.h>
+using namespace std;
+
+
+int t[9][9]; // for memorization
+
+int rodcut(int price[], int length[],
+                    int Max_len, int n)
+{
+    if (n == 0 || Max_len == 0)
+    {
+        return 0;
+    }
+    if (length[n - 1] <= Max_len)
+    {
+        t[n][Max_len]  = max(price[n - 1]+ rodcut(price, length, Max_len - length[n - 1], n),rodcut(price, length, Max_len, n - 1));
+    }
+
+    else
+    {
+        t[n][Max_len]= rodcut(price, length, Max_len, n - 1);
+    }
+
+    return t[n][Max_len];
+}
+
+/* Driver program to
+test above functions */
+
+int main()
+ {
+
```

Comment:
indent your code properly, remove extra blank lines.

Reply:
Done ,please review the changes in new commit